test/images/agnhost/guestbook/guestbook.go

```diff
@@ -77,16 +77,18 @@ func registerNode(registerTo, port string) {
 	}
 
 	hostPort := net.JoinHostPort(registerTo, backendPort)
-	_, err := net.ResolveTCPAddr("tcp", hostPort)
-	if err != nil {
-		log.Fatalf("--slaveof param and/or --backend-port param are invalid. %v", err)
-		return
-	}
 
 	request := fmt.Sprintf("register?host=%s", getIP(hostPort).String())
 	log.Printf("Registering to master: %s/%s", hostPort, request)
 	start := time.Now()
 	for time.Since(start) < timeout {
+		_, err := net.ResolveTCPAddr("tcp", hostPort)
+		if err != nil {
+			log.Printf("--slaveof param and/or --backend-port param are invalid. %v. Retrying in 1 second.", err)
```

Comment:
I think that the message should reflect better the error, maybe `unable to resolve %s --slaveof param and/or --backend-port param with error %v. Retrying in second`?

Reply:
I feel its better to print the value of a second than hardcoded value.

Reply:
Will make both of these updates :+1: